packages/evm/src/opcodes/functions.ts

```diff
@@ -36,6 +36,16 @@ import type { Common } from '@ethereumjs/common'
 
 const EIP3074MAGIC = hexToBytes('0x03')
 
+const BIGINT_0 = BigInt(0)
+const BIGINT_1 = BigInt(1)
+const BIGINT_2 = BigInt(2)
```

Comment:
For constants other than `0` and `1`, could you add a comment why these are relevant?